nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -181,48 +181,45 @@ def _set_backend_entity(self, model: TModel) -> None:
                 OVMinMaxAlgoBackend
             self._backend_entity = OVMinMaxAlgoBackend()
+        elif model_backend == BackendType.TORCH:
+            from nncf.quantization.algorithms.min_max.torch_backend import \
+                PTMinMaxAlgoBackend
+            self._backend_entity = PTMinMaxAlgoBackend()
         else:
             raise RuntimeError('Cannot return backend-specific entity '
                                'because {} is not supported!'.format(model_backend))
 
-    def _get_default_statistics_collector(self, is_symmetric: bool,
-                                          axes: Optional[Tuple], per_channel: bool) -> TensorStatisticCollectorBase:
-        """
-        Returns the default StatisticCollector.
-
-        :param is_symmetric: True if the quantizer has symmetric mode. False if asymmetric.
-        :param axes: Axes to reduce in the statistic tensor.
-        :param per_channel: True if per-channel statistics. False if per-tensor.
-        :return: StatisticCollector.
-        """
-        if per_channel:
-            return self._backend_entity.minmax_statistic_collector(use_abs_max=is_symmetric,
-                                                                   reduction_shape=axes,
-                                                                   num_samples=self._parameters.number_samples)
-        return self._backend_entity.mean_minmax_statistic_collector(use_per_sample_stats=False,
-                                                                    use_abs_max=is_symmetric,
-                                                                    reduction_shape=axes,
-                                                                    num_samples=self._parameters.number_samples)
-
-    def _get_stat_collector(self, quantizer_config: QuantizerConfig) -> TensorStatisticCollectorBase:
+    def _get_stat_collector(self, nncf_graph: NNCFGraph,
+                            target_point: TargetPoint,
+                            quantizer_config: QuantizerConfig) -> TensorStatisticCollectorBase:
         """
         Creates and returns statistic collector instance based on the quantizer's configuration.
 
         :param quantizer_config: QuantizerConfig instance for the current layer.
         :return: One of the TensorStatisticCollectorBase instances
         """
-        is_symmetric = quantizer_config.mode == QuantizationMode.SYMMETRIC
-        axes = (0, 2, 3) if quantizer_config.per_channel else None
-        if self._parameters.range_type is None or quantizer_config.per_channel:
-            return self._get_default_statistics_collector(is_symmetric, axes, quantizer_config.per_channel)
-        if self._parameters.range_type == RangeType.MINMAX:
-            return self._backend_entity.minmax_statistic_collector(use_abs_max=is_symmetric,
-                                                                   reduction_shape=axes,
+        def is_default_parameters(range_type: RangeType,
```

Comment:
To be honest, I don't see any other place (only a few lines below) where is this method used. So, maybe it's not needed?

Reply:
This is made as a response on this comment https://github.com/openvinotoolkit/nncf/pull/1471#discussion_r1108674450
@kshpv, let's choose best option of default params describing

Reply:
This structure helps to understand the code and add extensibility for the adding functionlaity